designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,51 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+This is the “all-projects” view which has been modified to now expose more information about the namespaces and without listing out every CSV. The goal of this view is to consolidate the list of operators. There are a few additions to the flow:
+
+1. We have introduced the Operator Namespace column. The Operator Namespace column has 2 descriptors that can live in each row; the first is a single project like “my-project” that is associated with an operator like 3scale. If a namespace appears across All Namespaces it will appear as All Namespaces. In the example of an operator like Jaeger, the text underneath has the namespace name and All Namespaces underneath.
+
+2. In the Managed Namespace column there are three different ways the Namespace information can appear. “My-project” is representative of a single namespace. “All namespaces” is a text only link that is associated with the namespaces that fall across all Namespaces. Thirdly, there is a multi-project view. For example, the Portworx operator falls across multiple projects but not all projects, so in this example the Managed Namespace column will have a popover on click. The namespace names should be listed underneath as seen in the mockup.
+
+![all-projects-view](img/all-projects-view.jpg)
+
+Clicking into the 2 namespaces field, the popover will appear:
+![popover-all-projects-view](img/popover-all-projects-view.jpg)
+
+## Single Project view
+
+For the first example, 3Scale, when a user clicks into an operator that is associated with a single namespace, on the Operator Details page, the Namespace will read Managed Namespace. This is how this view will look on a single-project view.
+![my-project-view](img/3scale-my-project-view.jpg)
+
+When the user selects the 3scale operator, we land on the Operator Details page:
+![3scale-operator-details](img/3scale-operator-details.jpg)
+
+Here is the in-line Managed Namespace field:
+![3scale-my-project-detail](img/3scale-my-project-detail.jpg)
+
+## Subscription page
+
+On the Subscription page, we’ve updated the copy to say Managed Namespace.
+![3scale-subscription](img/3scale-subscription.jpg)
+
+## Select Operator in All Namespaces view
+
+User selects Jaeger Operator:
+![img/Jaeger-my-project-view](img/Jaeger-my-project-view.jpg)
+
+In the second example, when a user clicks into the Jaeger Operator they will be taken to the Operator Details page, and the Namespace field should read Managed Namespace.
+![Jaeger-details](img/Jaeger-details.jpg)
```

Comment:
@rachael-phillips @tlwu2013 Would the project selector in this details view be the operator namespace as shown, or the managed namespace that was in the project selector in the list view when clicked ("my-project" in this case?) I think there was concern the user might not have access to the operator namespace and only their managed namespace?

Maybe if the operator is opened from the 'all projects' list view they are taken to the operator namespace operator details, and if from a single namespace they are taken to that namespace operator details?

Reply:
What do you think @tlwu2013?

Reply:
The project selector on the details view should be the **managed namespace**, i.e. the project selector in the list view when clicked ("_my-project_" in this case).

As for coming from 'all projects' list view, I second your idea that users are taken to the operator namespace operator details (i.e. in "openshift-operators" NS); if from a single namespace, they are taken to that namespace operator details.

Reply:
To confirm @itsptk and @tlwu2013, when a user selects "all projects" from the project selector on the Installed Operators page and then the user clicks into the 3scale operator, the user is taken to the 3scale operator details page, and the field should read "Operator Namespace" rather than Managed Namespace?

Reply:
In the 3Scale example, on the all projects view, when the user clicks in the project-selector will say, my project. Here is the image with the changes made: 
![3scale-all-projects-operator-details](https://user-images.githubusercontent.com/40275806/77193365-823e4d00-6ab4-11ea-8cb2-5da1f0cb1506.jpg)

Reply:
Actually, that comment was referring to the value of the project selector when viewing the operator details page. The ClusterServiceVersion Overview would always show the "Managed Namespace" and the Subscription Overview would always show the "Operator Namespace."

The thought is that if the user clicked on an operator in the list view while the project selector is a single namespace, the details view would also be that same single namespace. If the user clicked on an operator in the list view while the project selector is 'all-namespaces,' the details view would have the project selector = to that operator's "operator namespace" since we wouldn't have the context of a single managed namespace to use.

Reply:
Ok thank you so much, @itsptk - that's very clear and I can make these changes!